static/skin/index.js

```diff
@@ -1,71 +1,75 @@
+    }
 
-function generateBookHtml(book) {
-    const link = book.querySelector('link').getAttribute('href');
-    const title =  getInnerHtml(book, 'title');
-    const description = getInnerHtml(book, 'summary');
-    const id = getInnerHtml(book, 'id');
-    const iconUrl = getInnerHtml(book, 'icon');
-    const articleCount = getInnerHtml(book, 'articleCount');
-    const mediaCount = getInnerHtml(book, 'mediaCount');
+    function generateBookHtml(book) {
+        const link = book.querySelector('link').getAttribute('href');
+        const title =  getInnerHtml(book, 'title');
+        const description = getInnerHtml(book, 'summary');
+        const id = getInnerHtml(book, 'id');
+        const iconUrl = getInnerHtml(book, 'icon');
+        const articleCount = getInnerHtml(book, 'articleCount');
+        const mediaCount = getInnerHtml(book, 'mediaCount');
 
-    return `<a href='${link}' data-id='${id}'><div class='book'>
-        <div class='book__background' style="background-image: url('${iconUrl}');">
-        <div class='book__title' title='${title}'>${title}</div>
-        <div class='book__description' title='${description}'>${description}</div>
-        <div class='book__info'>${articleCount} articles, ${mediaCount} medias</div>
-        </div>
-    </div></a>`;
-}
+        return `<a href='${link}' data-id='${id}'><div class='book'>
+            <div class='book__background' style="background-image: url('${iconUrl}');">
+            <div class='book__title' title='${title}'>${title}</div>
+            <div class='book__description' title='${description}'>${description}</div>
+            <div class='book__info'>${articleCount} articles, ${mediaCount} medias</div>
+            </div>
+        </div></a>`;
+    }
 
-async function loadAndDisplayBooks() {
-    isFetching = true;
-    fetch(queryUrlBuilder()).then(async (resp) => {
-        const data = new window.DOMParser().parseFromString(await resp.text(), 'application/xml');
-        const books = data.querySelectorAll('entry');
-        let bookHtml = '';
-        books.forEach((book) => {bookHtml += generateBookHtml(book)});
-        document.querySelector('.book__list').innerHTML += bookHtml;
-        incrementalLoadingParams['start'] = incrementalLoadingParams['start'] + incrementalLoadingParams['count'];
-        if (!books.length) {
-            incrementalLoadingParams['count'] = 0;
+    async function loadAndDisplayBooks() {
+        isFetching = true;
+        fetch(queryUrlBuilder()).then(async (resp) => {
+            const data = new window.DOMParser().parseFromString(await resp.text(), 'application/xml');
+            const books = data.querySelectorAll('entry');
+            let bookHtml = '';
+            books.forEach((book) => {bookHtml += generateBookHtml(book)});
+            document.querySelector('.book__list').innerHTML += bookHtml;
+            incrementalLoadingParams.start += incrementalLoadingParams.count;
```

Comment:
Currently, it doesn't make any difference, but it's more future-proof to advance `incrementalLoadingParams.start` by the count of items actually loaded.